tests/Event/ReadModel/RDF/EventJsonToTurtleConverterTest.php

```diff
@@ -92,6 +95,7 @@ protected function setUp(): void
             $this->documentRepository,
             (new EventDenormalizer())->handlesDummyOrganizers(),
             $addressParser,
+            new RdfResourceFactoryWithoutBlankNodes(new NodeUriGenerator(new CRC32HashGenerator())),
```

Comment:
Maybe this needs to be added to the container so it can be reused.

Reply:
See my remark about this being a test in the other pr. I don't think we ever use the container for unit tests?